sympy/geometry/parabola.py

```diff
@@ -376,6 +376,21 @@ def p_parameter(self):
             p = sign(self.focus.args[1] + y)
         return p * self.focal_length
 
+    @property
+    def latus_rectum(self):
+        """Returns the line passing through the focus that is perpendicular to the axis of symmetry
+
```

Comment:
Break long lines into small ones. Each line should be of less than 72 characters, using PEP8